### PR TITLE
INK2-764-library-changes

### PR DESCRIPTION
**Pull request by arora01gunjan**

### CODE REVIEW

File: fetch_response.go [hunk 1]
```diff
@@ -3,6 +3,8 @@ package sarama
 import (
 	"sort"
 	"time"
+
+	"bitbucket.org/ffxblue/bp-gateway/pkg/monolog"
```
Comment (dackroyd) [inline fetch_response.go:7]: Internal project packaging, unrelated to Sarama, Need to remove all internal code references/use

File: fetch_response.go [hunk 2]
```diff
@@ -450,7 +452,15 @@ func (r *FetchResponse) AddRecordWithTimestamp(topic string, partition int32, ke
 		frb.RecordsSet = []*Records{&records}
 	}
 	batch := frb.RecordsSet[0].RecordBatch
-	rec := &Record{Key: kb, Value: vb, OffsetDelta: offset, TimestampDelta: timestamp.Sub(batch.FirstTimestamp)}
```
Comment (dackroyd) [inline fetch_response.go:453]: Fundamentally the issue here is that it needs to be possible to include headers in the record, not these specific ones?

**Review by dackroyd — CHANGES_REQUESTED** [review]

**Comment by dackroyd** [comment]
It's likely that we can keep these changes within the fork, without a PR back to the main repo for now

**Comment by dnwe** [comment]
From the comments above it sounds like this was an accidentally raised PR containing some code to set a number of default record headers based on a private company project?

Will go ahead and close this PR for now but feel free to come back if you think I'm mistaken :)

**Pull request closed without merging**